Assets/Scripts/Controllers/JobSpriteController.cs

```diff
@@ -68,7 +68,7 @@ void OnJobCreated(Job job)
             //until the graphics system for tiles is fleshed out further
 
             job_go.transform.position = new Vector3(job.tile.X, job.tile.Y, 0);
-            sr.sprite = SpriteManager.current.GetSprite("Tile", "Empty");
+            sr.sprite = SpriteManager.current.GetSprite("Tile", "Solid");
```

Comment:
Is the `Solid` one 100% transparent? Also I don't see the need to change the name as technically that should still be empty space.